### PR TITLE
Mat 6168 edit definition

### PR DESCRIPTION
**Pull request by sb-cecilialiu**
## MADiE PR

Jira Ticket: [MAT-6168](https://jira.cms.gov/browse/MAT-6168)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [ ] Required tests are included.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [x] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/components/editMeasure/details/MeasureMetaDataRow.tsx [hunk 1]
```diff
@@ -10,10 +10,11 @@ interface MeasureMetaDataRowProps {
   description: string;
   id?: string;
   handleEdit?: Function;
+  canEdit?: boolean;
 }
 
 const MeasureMetaDataRow = (props: MeasureMetaDataRowProps) => {
-  const { name, description, id, handleEdit } = props;
+  const { name, description, id, handleEdit, canEdit } = props;
```
Comment (RohitKandimalla) [inline src/components/editMeasure/details/MeasureMetaDataRow.tsx:16]: 1. Looks like we have unused imports
2. onClick is taking a return object "e" which is never used, may be we can remove it as well.

Comment (sb-cecilialiu) [reply src/components/editMeasure/details/MeasureMetaDataRow.tsx:16]: @RohitKandimalla Thanks I removed those two.

Comment (RohitKandimalla) [reply src/components/editMeasure/details/MeasureMetaDataRow.tsx:16]: I am little confused on the implementation, Help me understand. Why does References are not editable ?
MeasureReference & MeasureDefinition both are sending a key prop to MeasureMetaDataRow but is never captured in child component. 

If we haven't worked on Editing references yet, then may be we can use "key" prop as intended or remove it

Comment (sb-cecilialiu) [reply src/components/editMeasure/details/MeasureMetaDataRow.tsx:16]: @RohitKandimalla I just checked the code, looks like it's implemented but behind a flag, so i added the canEdit for references as well.

Comment (RohitKandimalla) [reply src/components/editMeasure/details/MeasureMetaDataRow.tsx:16]: Not the canEdit, The prop "key" 
MeasureMetaDataRowProps doesn't have a prop "key" but both the parent components are sending a prop "key" do we need it ?

Comment (sb-cecilialiu) [reply src/components/editMeasure/details/MeasureMetaDataRow.tsx:16]: @RohitKandimalla I thought for a list we do need a key?

Comment (RohitKandimalla) [reply src/components/editMeasure/details/MeasureMetaDataRow.tsx:16]: we do but its never used

Comment (sb-cecilialiu) [reply src/components/editMeasure/details/MeasureMetaDataRow.tsx:16]: @RohitKandimalla ok i removed it. thanks

**Review by RohitKandimalla — APPROVED** [review]

**Pull request merged**